resources/darwin/bin/nerdctl

```diff
@@ -2,11 +2,11 @@
 set -eu -o pipefail
 
 scriptdir="${BASH_SOURCE[0]}"
-[ -L "${scriptdir}" ] && scriptname="$(readlink "${scriptdir}")"
-scriptdir="$(cd "$(dirname "${scriptname}")" && pwd)"
+[ -L "${scriptdir}" ] && scriptdir="$(readlink "${scriptdir}")"
+scriptdir="$(cd "$(dirname "${scriptdir}")" && pwd)"
```

Comment:
It might make sense to have two variables here:
```suggestion
scriptname="${BASH_SOURCE[0]}"
[ -L "${scriptname}" ] && scriptname="$(readlink "${scriptname}")"
scriptdir="$(cd "$(dirname "${scriptname}")" && pwd)"
```

Do we need to worry about multiple layers of symlinks for the initial script?

Reply:
Thanks! Using different names is certainly clearer. Not sure if multiple symlink layers are a concern (we should only create a single symlink via the UI), but dealing with it is trivial, so why not?